src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf); font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf); font-style: normal;
+  font-weight: 800;
+}
+
+.search {
+  margin-top: 20px;
+  overflow: hidden;
+  position: relative;    
+}
+
+.input::placeholder {
+  font-weight: 300;
+}
+  
+.input {
+  font-weight: 800;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  height: 70px;
+  padding-left: 70px;
+  font-family: "Avenir", Arial, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+} 
+
+.input--small {    
+  font-weight: 500;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+} 
+  
+.search:hover {
+  box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+  
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%,  #f6f6f7 100%);
+  box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 3px;
+}
+        
+.icon {
+  width: 19px;
+  z-index: 1;
+  position: absolute;
+  left: 26px;
+  top: 26px;
+}
+
+.icon--small {
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
```

Comment:
Add newline at the end
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file